__tests__/newrelic.tests.js

```diff
@@ -0,0 +1,12 @@
+const newrelic = require( '../src/newrelic/' );
+
+test( 'New Relic module should fail if no license key is given', () => {
+	// eslint-disable-next-line no-undef
+	const spyOnError = jest.spyOn( console, 'error' );
+
+	newrelic();
+
+	expect( spyOnError ).toHaveBeenCalled();
+
+	spyOnError.mockRestore();
+} );
```

Comment:
For the "success" test case, we cna probably use jest's auto-mock support: https://jestjs.io/docs/en/manual-mocks#mocking-node-modules

Basically, add a path called `__mocks__/newrelic.js` and we have it export a value we can verify in our test.

Reply:
Never heard of the auto mocking, that's actually very helpful. Thanks 🙌